lib/pages/chat/chat_view.dart

```diff
@@ -124,16 +114,11 @@ class ChatView extends StatelessWidget with MessageContentMixin {
 
   @override
   Widget build(BuildContext context) {
-    controller.matrix ??= Matrix.of(context);
-    final client = controller.matrix!.client;
-    controller.sendingClient ??= client;
-    controller.room = controller.sendingClient!.getRoomById(controller.roomId!);
+    controller.initRoom();
```

Comment:
we can not do it in better way, keep it as current status.

Reply:
reverted